cmd/controller-manager/main.go

```diff
@@ -186,7 +191,7 @@ func main() {
 			Lock: &resourcelock.EndpointsLock{
 				EndpointsMeta: metav1.ObjectMeta{
 					Namespace: ns,
-					Name:      "tidb-controller-manager",
+					Name:      "tidb-controller-manager-" + helmRelease,
```

Comment:
There are compatibility issues here when upgrade operator.

Reply:
@mikechengwei If `helmRelease` is set, a new endpoint with suffix will be created, and the old one won't be deleted, both of them can work normally. In which situation do you think will cause compatibility issues? I don't get it , please explain with more detail, thanks a lot 😄

Reply:
There is nothing wrong with this piece of code, just set the default value, compatible with the old logic, the new commit should be no problem.

Reply:
Got it